software/messaging/src/main/java/brooklyn/entity/messaging/rabbit/RabbitSshDriver.java

```diff
@@ -186,6 +195,11 @@ public void kill() {
                 .put("RABBITMQ_NODENAME", getEntity().getId())
                 .put("RABBITMQ_NODE_PORT", getAmqpPort().toString())
                 .put("RABBITMQ_PID_FILE", getRunDir()+"/"+getPidFile())
+                .put("RABBITMQ_CONFIG_FILE", getConfigPath())
                 .build();
     }
+
+    private String getConfigPath() {
+        return getInstallDir() + "/rabbitmq";
```

Comment:
That's because you're doing it before executing the main script. Using `newScript(CUSTOMIZE)` magically creates runDir if it doesn't already exist.